docs/appendices/types.rst

```diff
@@ -57,6 +57,17 @@ records.
 
 .. _types-caa:
 
+APL
+-----
+
+.. versionadded:: tbd
```

Comment:
@Habbie check-spelling will complain about this `tbd`.
the pdns repository shows no sign of a `versionadded` field ever having `tbd` (or any non number, the closest was `4.2.x` which appears to be an error).

Reply:
I'll set this to 4.4.0; please let me know if not correct.

Reply:
4.4.0 is a decent assumption!

Reply:
Done, will update if a change is needed.